recipes/onetbb/all/conanfile.py

```diff
@@ -56,8 +56,9 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def package_id(self):
-        if Version(self.version) < "2021.6.0":
+        if Version(self.version) < "2021.5.0":
             del self.info.options.tbbmalloc
+        if Version(self.version) < "2021.6.0":
             del self.info.options.tbbproxy
```

Comment:
I think there is something wrong here. tbbmaloc & tbbproxy should be removed in config_options depending on versions.

Reply:
the problem is that TBB had tbbmalloc and tbbproxy for a while, but only recently has provided options to disable them.
So, I suppose it's correct to ignore these options in `package_id` only